### PR TITLE
Updating deprecation warning for ServiceOps to use cats semigroup inst…

### PR DESCRIPTION
**Pull request by calvinbrown085**
…ead of scalaz semigroup

https://github.com/http4s/http4s/issues/1502

### CODE REVIEW

File: server/src/main/scala/org/http4s/server/syntax/package.scala [hunk 1]
```diff
@@ -6,7 +6,7 @@ import cats.data.Kleisli
 import cats.implicits._
 
 package object syntax {
-  @deprecated("Import `cats.syntax.semigroup._` and use `service1 |+| service2` instead", "0.16")
+  @deprecated("Import `cats.implicits._` and `org.http4s.implicits._` and use `service1 <+> service2` instead. Ensure -Ypartial-unification is enabled.", "0.16")
```
Comment (rossabaker) [inline server/src/main/scala/org/http4s/server/syntax/package.scala]: `org.http4s.implicits._` used to be required, but is not as of 0.18.0-M5.  (The semigroupk for kleisli that we got from it is in cats-1.0.0-RC1).

**Comment by calvinbrown085** [comment]
@rossabaker Ah, okay. Fixed that.

**Comment by aeons** [comment]
Travis failed due to formatting.

Run `;scalafmt;test:scalafmt` in sbt to format everything.

You can check whether a certain commit will (probably) pass CI by running `sbt ci` on your local machine.

**Comment by calvinbrown085** [comment]
@aeons Ah, cool I didn't know that existed. Formatting should be fixed now.

**Comment by aeons** [comment]
Too long lines :)

**Pull request merged**